src/main/java/de/slimecloud/slimeball/features/staff/absence/AbsenceTable.java

```diff
@@ -0,0 +1,30 @@
+package de.slimecloud.slimeball.features.staff.absence;
+
+import de.mineking.databaseutils.Table;
+import de.mineking.databaseutils.Where;
+import de.mineking.discordutils.list.Listable;
+import net.dv8tion.jda.api.entities.UserSnowflake;
+
+import java.time.Instant;
+import java.util.List;
+import java.util.Optional;
+
+public interface AbsenceTable extends Table<Absence>, Listable<Absence> {
+
+	default Absence addAbsence(Absence absence) {
+		insert(absence);
+		return absence;
+	}
+
+	default void remove(Absence absence) {
+		delete(absence);
+	}
+
+	default List<Absence> expiredAbsence(Instant now) {
```

Comment:
Missing @NotNull for return and parameter. Also the method name should contain `get`